unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/EntityIdHelper.cs

```diff
@@ -35,12 +39,57 @@ public long EntityFromLegacyEntityString(string entityId)
                     return (long) SpecialEntityId.THIRD_PERSON_CAMERA_ENTITY_REFERENCE;
             }
 
-            long entityIdLong = entityId.GetHashCode() << 9;
-
+            long entityIdLong = DecodeBase36(entityId) << 9;
+            
             if (!entityIdToLegacyId.ContainsKey(entityIdLong))
                 entityIdToLegacyId[entityIdLong] = entityId;
 
             return entityIdLong;
         }
+        
+        public Int64 DecodeBase36(string input)
```

Comment:
Why are we using `Int64` here instead of `long`?

I know that they are almost the same, but we should have consistency with the type name.

Reply:
Just a copy paste mistake!